Tiltfile

```diff
@@ -169,7 +212,7 @@ WORKDIR /
 COPY --from=tilt-helper /start.sh .
 COPY --from=tilt-helper /restart.sh .
 COPY --from=tilt-helper /go/bin/dlv .
-COPY manager .
+COPY $binary_name .
```

Comment:
Don't you also need a `ARG binary_name` in this section of the Dockerfile?

Reply:
Apparently not - this works fine right now and is implemented AFAIK by the `docker_build()` function in Tilt

Reply:
Ah interesting!